experiments/verifier_exposure/run.py

```
@@ -0,0 +1,1135 @@
+        shutil.rmtree(workspace_dir)
+    if runtime_dir.exists():
+        shutil.rmtree(runtime_dir)
+    output_dir.mkdir(parents=True, exist_ok=True)
+    runtime_dir.mkdir(parents=True, exist_ok=True)
+    roots = _prepare_roots(workspace_dir, runtime_dir, output_dir)
+    identity = _build_container_identity(runtime_dir)
+    assembled = _assemble_workspace(item, roots)
+    runtime = load_runtime("base")
+    cmd = _build_docker_command(item, runtime, roots, identity, interactive=True)
+    start = datetime.now(timezone.utc)
+    exit_code = subprocess.run(cmd, check=False).returncode
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)